server/controllers/review.js

```diff
@@ -0,0 +1,49 @@
+const Review = require("../models/Review");
+
+// @route GET /reviews/all/:sitterId
+// @desc gets all reviews for sitter
+// @access Private
+exports.getAllReviews = async (req, res) => {
+  const reviews = await Review.find({ sitterId: req.params.sitterId });
+  res.json({ success: reviews });
+};
+
+// @route POST /reviews/create
+// @desc creates new review
+// @access Private
+exports.createReview = async (req, res) => {
+  const { sitterId, rating, description } = req.body;
+
+  const sitter = await User.findById(sitterId);
+  const sitterProfile = await Profile.findById(sitter.profile);
+
+  if (!sitterProfile.isSitter) {
+    res.json({ error: "Supplied sitter is not a sitter (this should be true)" });
+  }
+
+  res.send({ success: req.user.id });
+
+  const client = await User.findById(req.user.id);
```

Comment:
Since the User model contains a ref to the profile, why don't we just populate the profile here rather than querying the Profile model in the next line?